public/index.html

```diff
@@ -8,6 +8,10 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <link rel="stylesheet" href="https://muffinui.netlify.app/style.css" />
     <link rel="stylesheet" href="../src/App.css" />
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/mockman-js@latest/dist/style.css"
+    />
```

Comment:
Give every link to a proper comment so that everyone can know.

Reply:
Yes will add comment